packages/rule-tester/src/RuleTester.ts

```diff
@@ -115,11 +161,16 @@ function getUnsubstitutedMessagePlaceholders(
 }
 
 export class RuleTester extends TestFramework {
+  // instead of creating a hard dependency, just use a soft require
+  // a bit weird, but if they're using this tooling, it'll be installed
+  // eslint-disable-next-line @typescript-eslint/no-require-imports
+  readonly #defaultParser = require(
+    TYPESCRIPT_ESLINT_PARSER,
+  ) as typeof ParserType;
```

Comment:
we've got this `require` plus the `require` up on L72.
we should extract it to a global and reuse the `require`.